code/datums/json_savefile.dm

```diff
@@ -11,7 +11,7 @@
 	/// Cooldown that tracks the time between attempts to download the savefile.
 	COOLDOWN_DECLARE(download_cooldown)
 
-GENERAL_PROTECT_DATUM(/datum/json_savefile)
```

Comment:
Make sure you don't accidentally merge with this in.

Reply:
i fucking hate this, i honestly want it gone